lib/internal/validators.js

```diff
@@ -258,6 +258,17 @@ function validateUnion(value, name, union) {
   }
 }
 
+function validateLinkHeaderValue(value, name) {
+  const linkValueRegExp = /((<(.*)>;) (rel=(.*);) (as=(.*)))/;
```

Comment:
Not exhaustive suggestion, still has room for spec-compliance improvement
```suggestion
  const linkValueRegExp = /((<([^>]*)>;)\s*(rel="?([^;]*)"?;)\s*(as="?([^\s]*)"?))/;
```

Reply:
thanks! any help on improving regex is always welcome 😅